assets/js/admin/settings.js

```diff
@@ -161,12 +161,26 @@
 				lastRow  = $( this ).find( 'tbody tr:last' ),
 				firstRow = $( this ).find( 'tbody tr:first' );
 
-			table.find( '.wc-item-reorder-nav .wc-move-disabled' ).removeClass( 'wc-move-disabled' ).attr( { 'tabindex': '0', 'aria-hidden': 'false' } );
-			firstRow.find( '.wc-item-reorder-nav .wc-move-up' ).addClass( 'wc-move-disabled' ).attr( { 'tabindex': '-1', 'aria-hidden': 'true' } );
-			lastRow.find( '.wc-item-reorder-nav .wc-move-down' ).addClass( 'wc-move-disabled' ).attr( { 'tabindex': '-1', 'aria-hidden': 'true' } );
+			table.find( '.wc-item-reorder-nav .wc-move-disabled' ).removeClass( 'wc-move-disabled' )
+				.attr( { 'tabindex': '0', 'aria-hidden': 'false' } );
+			firstRow.find( '.wc-item-reorder-nav .wc-move-up' ).addClass( 'wc-move-disabled' )
+				.attr( { 'tabindex': '-1', 'aria-hidden': 'true' } );
+			lastRow.find( '.wc-item-reorder-nav .wc-move-down' ).addClass( 'wc-move-disabled' )
+				.attr( { 'tabindex': '-1', 'aria-hidden': 'true' } );
 		} );
 
 		$( '.wc-item-reorder-nav').closest( 'table' ).trigger( 'updateMoveButtons' );
 
+
+		$( '.submit button' ).on( 'click', function() {
+			if ( $( 'select#woocommerce_allowed_countries' ).val() === 'specific'
+				&& ! $("[name='woocommerce_specific_allowed_countries[]']").val() ) {
```

Comment:
There 3 linter errors to fix for this file:

![Screenshot from 2019-05-02 19-16-46](https://user-images.githubusercontent.com/1264099/57110409-da3e5a80-6d0e-11e9-9beb-9f973088fd78.png)

Reply:
Thanks, seems like I need to update my linter rules.